### DISCUSSION TITLE
Welcome to nohang Discussions!

### DISCUSSION THREAD

**Discussion by hakavlad** [hakavlad/nohang#111] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexmyczko** [comment]
If you wanted an icon for it? What would it look like?

Here's my proposition:

![nohang](https://user-images.githubusercontent.com/21078693/229369631-5a05acb7-18fd-493c-aab7-a199ad02adc3.png)

**Reply by hakavlad** [reply]
@alexmyczko Thank you for the suggestion. I don't think nohang needs an icon: it's not a GUI application.

**Comment by FabioPedretti** [comment]
Hi, do you plan for a new release? There are some fixes accumulated. For example Ubuntu has this issue: https://bugs.launchpad.net/ubuntu/+source/nohang/+bug/2049206 which is already fixed in git.
Thanks!

**Reply by hakavlad** [reply]
@FabioPedretti Hi, I plan to do a new release. I hope soon.
